src/sql/workbench/services/connection/browser/connectionDialogService.ts

```diff
@@ -330,7 +333,6 @@ export class ConnectionDialogService implements IConnectionDialogService {
 		let providerName = model ? model.providerName : defaultProvider;
 		providerName = providerName ? providerName : defaultProvider;
 		let newProfile = new ConnectionProfile(this._capabilitiesService, model || providerName);
-		newProfile.saveProfile = true;
```

Comment:
Why are you removing this? That's going to change existing behavior for connections right?